docs/source/using/addons.adoc

```diff
@@ -92,7 +92,7 @@ The `oc` command is executed as *system:admin*.
 ====
 
 openshift::
-If the add-on command starts with `openshift`, you can run the *openshift* binary within the container that runs OpenShift.
+If the add-on command starts with `openshift`, you can run the *oc* binary within the container that runs OpenShift.
```

Comment:
👍 . At least we are transparent now about what we are doing.

Reply:
This will create confusion which one to use. `oc adm` or `openshift adm`.

We need to give more info about when to use which command.
![screenshot from 2018-03-27 11-36-36](https://user-images.githubusercontent.com/1132451/37949711-6624bde8-31b3-11e8-9550-742fa0706c00.png)

Reply:
@praveenkumar Here is my suggestion:
If the add-on command starts with `openshift`, it uses the `oc` binary that is present in the OpenShift container.
This means that any file parameters or other system-specific parameters must match the environment of the container instead of your host. 
This command should be used when successful execution of the `oc` command needs to change something in the OpenShift container.

Reply:
well, that was the suggestion I made; `openshift` should indicate a remote `oc` instance; in this case happening inside the origin server container. while `oc` will start a local `oc` from `OcPath`.

Reply:
Another attempt: 

If the add-on command starts with openshift, it uses the oc binary present in the OpenShift container to execute the command.
This is useful when the oc command required to be inside the OpenShift container for successful execution of the add-on command. 
This means that any file parameters or other system-specific parameters must match the environment of the container instead of your host.